### PR TITLE
remotedesktop - fix installation on Ubuntu 22.04 and higher

### PR DESCRIPTION
**Pull request by szaimen**

### CODE REVIEW

File: not-supported/remotedesktop.sh [hunk 1]
```diff
@@ -329,34 +320,6 @@ We will need to add a 3rd party repository to install it which can set your serv
...
-            print_text_in_color "$ICyan" "$SUBTITLE was successfully uninstalled."
-        else
-            msg_box "OnlyOffice Desktop Editors are open source but not existing in the Ubuntu repositories.
-Hence, we will add a 3rd-party repository to your server \
-to be able to install and update OnlyOffice Desktop Editors using the apt packet manager.
-This can set your server under risk, though!" "$SUBTITLE"
-            if yesno_box_yes "Do you want to install OnlyOffice Desktop Editors nonetheless?" "$SUBTITLE"
-            then
-                print_text_in_color "$ICyan" "Installing $SUBTITLE"
-                apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys CB2DE8E5
```
Comment (szaimen) [inline not-supported/remotedesktop.sh:350]: maybe adding the key in recommended way via add_trusted_key_and_repo can fix this.

Comment (enoch85) [reply not-supported/remotedesktop.sh:350]: You should use that function no matter what. The behavior is updated in 22.04, hence the function.

**Comment by enoch85** [comment]
@szaimen Is this still WIP?

**Comment by szaimen** [comment]
> @szaimen Is this still WIP?

yes, didnt get to it yet...

**Comment by szaimen** [comment]
This is finally ready now

**Pull request merged**